src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,107 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+/**
+ * The class that handles the storage of the task list.
+ * It reads from the file and writes to the file.
+ * It also creates the file if it does not exist.
+ * The file is stored in the data folder.
+ */
+public class Storage {
+
+    private String filePath = "data/duke.txt";
```

Comment:
Can this be made a constant "`private static final`"?

Reply:
Agreed